### PR TITLE
[7543] extends report serializer

### PR DESCRIPTION
**Pull request by hklarner**

### CODE REVIEW

File: adhocracy4/reports/serializers.py [hunk 1]
```diff
@@ -1,9 +1,24 @@
 from rest_framework import serializers
 
+from adhocracy4.api.dates import get_datetime_display
+
 from .models import Report
 
 
 class ReportSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(required=False, source="creator.username")
+    created_at = serializers.SerializerMethodField(required=False)
```
Comment (goapunk) [inline adhocracy4/reports/serializers.py:10]: maybe just use `created` here (and below) as we also use that in other places (e.g. comments)

**Review by goapunk — CHANGES_REQUESTED** [review]
generally, it looks good to me. One small comment and it needs a changelog

File: adhocracy4/reports/serializers.py (hunk 1 above)
Comment (hklarner) [reply adhocracy4/reports/serializers.py]: done

File: adhocracy4/reports/serializers.py [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 from .models import Report
 
 
 class ReportSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(required=False, source="creator.username")
+    created = serializers.SerializerMethodField(required=False)
+
+    @staticmethod
+    def get_created(report):
+        return get_datetime_display(datetime=report.created)
```
Comment (m4ra) [inline adhocracy4/reports/serializers.py:14]: we don't need that, see my comment above.

**Review by m4ra — CHANGES_REQUESTED** [review]

File: adhocracy4/reports/serializers.py (hunk 2 above)
Comment (m4ra) [reply adhocracy4/reports/serializers.py:14]: oh, is it for returning the date in a specific format? if yes then it makes sense. But take the staticmethod decorator away, and replace `report` with `self`.

File: adhocracy4/reports/serializers.py [hunk 3]
```diff
@@ -1,9 +1,24 @@
 from rest_framework import serializers
 
+from adhocracy4.api.dates import get_datetime_display
+
 from .models import Report
 
 
 class ReportSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(required=False, source="creator.username")
+    created = serializers.SerializerMethodField(required=False)
```
[the remark this answers is not in the document]
Comment (hklarner) [reply adhocracy4/reports/serializers.py:10]: resolved (`created` needs to be optional)

File: adhocracy4/reports/serializers.py (hunk 2 above)
Comment (hklarner) [reply adhocracy4/reports/serializers.py:14]: done

**Review by m4ra — APPROVED** [review]

**Pull request merged**